generators/client/templates/vue/package.json.ejs

```diff
@@ -129,7 +129,8 @@ limitations under the License.
     "typescript": "3.2.4",
     "uglifyjs-webpack-plugin": "2.0.1",
     "url-loader": "1.1.2",
-    "vue-jest": "3.0.3",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
- do we need babel ?
- can you remove ^ to use a fixed version ?
- maybe order it alphabetically

Reply:
According to the [documentation](https://github.com/vuejs/vue-jest/tree/master#usage-with-babel-7) I also thought we don't need it, but it seems there is a little problem with peer [dependencies](https://github.com/vuejs/vue-jest/issues/160).

Sure the `^` is not required. Will also sort it alphabetically (put it there as it "belongs" to vue-jest).

Reply:
not sure if I understood correctly, but, it seems you don't have to as you are upgrading to use jest > `v24` and also don't see any `babel-jest` dependency here:
> If you use jest < 24.0.0 and babel-jest make sure to install babel-core@bridge

Reply:
ignore my last comment :smile:

Reply:
I guess you are likewise confused like I was when reading the sentence and looking at our issue :man_shrugging: